package.json

```diff
@@ -12,7 +12,8 @@
     "prepublish": "npm run build"
   },
   "peerDependencies": {
-    "react": ">=14.x"
+    "prop-types": ">=15.6.0",
+    "react": ">=15.6.0"
```

Comment:
we can keep `react` where it was. es6 classes have been supported since 0.13 if I'm not mistaken

Reply:
I think the bump was for the prop-types split. I forget the exact version that happened though.

Reply:
Yeah, wasn't entirely sure what to do here, mostly because `prop-types` only starts at `15.5.0`. Anyway, can revert the react requirement to `>=14.x`.

Reply:
I think it's fine as is, we'll probably want our package version for social-logos to be `2.0.0` when we finalize though.